src/main/java/core/basesyntax/controller/StorageServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Product;
+import java.util.List;
+
+public class StorageServiceImpl implements StorageService<Product> {
+    private StorageDao<Product> storageDao;
+
+    public StorageServiceImpl() {
+        this.storageDao = new StorageDaoImpl();
```

Comment:
```suggestion
    public StorageServiceImpl(StorageDao storageDao) {
        this.storageDao = storageDao;
```
let's use DI here, we will learn about it a bit later, in general it will make code more flexible